dandiapi/api/services/embargo/exceptions.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import status
+
+from dandiapi.api.services.exceptions import DandiException
+
+
+class AssetNotEmbargoed(DandiException):
+    http_status_code = status.HTTP_400_BAD_REQUEST
+    message = 'Only embargoed assets can be unembargoed.'
+
+
+class DandisetNotEmbargoed(DandiException):
+    http_status_code = status.HTTP_400_BAD_REQUEST
+    message = 'Dandiset not embargoed'
+
+
+class EmbargoedDandisetOwnerRequired(DandiException):
```

Comment:
I think we can just use `DandisetOwnerRequired` for now.

Reply:
There's an existing `DandisetOwnerRequired` in the asset service
https://github.com/dandi/dandi-archive/blob/6f2276d0c57e422008b7ebc80eceabde0a4827ba/dandiapi/api/services/asset/exceptions.py#L6-L7

so I was trying to avoid naming conflicts / confusion, but I'd also prefer that name. If you think it's not a concern I'll change it.

Reply:
Oh, I was suggesting we use that one. It's a little odd but I think this exception is really a global one that just happened to get used in the asset module first.

Reply:
Sounds good to me. Maybe if we ever have more shared exceptions in the future we can factor them out into a common exceptions module.